user-registration-app/src/app/views/registration-form/registration-form.component.ts

```diff
@@ -161,14 +161,19 @@ export class RegistrationFormComponent implements OnInit {
 
   ngOnInit() {
     this.route.data.subscribe(({ registration }) => {
-      this.registrationForm = registration;
-      this.registrationForm.firstHackathon = false;
-      this.registrationForm.eighteenBeforeEvent = null;
-      this.registrationForm.mlhcoc = null;
-      this.registrationForm.mlhdcp = null;
-      this.parsePhone(this.registrationForm.phone);
-      this.phoneNoUse = this.prettifiedPhone;
-      this.diet_restr = this.registrationForm.dietaryRestriction !== null;
+      this.registrationForm = new Registration();
+      //Data to keep
+      //TODO: make this into an actual function to be called
```

Comment:
The saint Kabir says: if you can do it tomorrow, do it today; if you can do it today, do it now! :P

Reply:
Consider me inspired, I guess I'll go do that now lmao